.evergreen.yml

```diff
@@ -396,6 +396,7 @@ buildvariants:
     - name: deploy_to_prod
       git_tag_only: true
       patchable: false
+      priority: 100
```

Comment:
I can't remember if this should be on the task definition or here where you have it, but you'll find out if it works when you merge it.

Reply:
This should work 👍